src/QMCWaveFunctions/lcao/LCAOrbitalBuilder.h

```diff
@@ -80,6 +83,9 @@ class LCAOrbitalBuilder : public SPOSetBuilder
   bool putFromXML(LCAOrbitalSet& spo, xmlNodePtr coeff_ptr);
   bool putFromH5(LCAOrbitalSet& spo, xmlNodePtr coeff_ptr);
   bool putPBCFromH5(LCAOrbitalSet& spo, xmlNodePtr coeff_ptr);
+  bool LoadCMatrix_cplx(LCAOrbitalSet& spo,hdf_archive hin, int neigs,int setVal,int norbs);
+  bool LoadCMatrix(LCAOrbitalSet& spo,hdf_archive hin, int neigs,int setVal,int norbs);
+  void EvalPhaseFactor(PosType twist);
```

Comment:
Please also rename EvalPhaseFactor to EvalPeriodicImagePhaseFactors.

Reply:
Done